src/GraphQL.php

```diff
@@ -510,12 +510,39 @@ protected function getSchemaConfiguration($schema)
 
         $schema = is_array($schema) ? $schema : $this->schemas[$schemaName];
 
+        return static::getNormalizedSchemaConfiguration($schema);
+    }
+
+    /**
+     * @return array<array>
+     */
+    public static function getNormalizedSchemasConfiguration(): array
+    {
+        return array_filter(array_map(function ($schema) {
+            try {
+                return static::getNormalizedSchemaConfiguration($schema);
+            } catch (SchemaNotFound $e) {
+                return null;
+            }
+        }, config('graphql.schemas')));
+    }
+
+    /**
+     * @param  mixed  $schema
+     * @return array<array>
+     */
+    protected static function getNormalizedSchemaConfiguration($schema): array
+    {
         if (! is_string($schema)) {
             return $schema;
         }
 
+        if (! class_exists($schema)) {
+            throw new SchemaNotFound('Schema class '.$schema.' not found.');
+        }
+
         /** @var ConfigConvertible $instance */
-        $instance = app()->make($schema);
+        $instance = new $schema();
```

Comment:
We should not change that, btw

Reply:
Agreed it would be nice to keep that. I don't know how necessary it is to have dependencies injected for this thing.
But I know removing it would make it a breaking change. 🤔 Bummer. It's only been there for a moment and I'm probably the only person that's tried to use the feature. 😆 

I changed this to do a regular `new`ing up of the class so that the app doesn't have to get booted.

Reply:
TBH I'm not so much about the breaking change, hardly imagine people jumped on this new feature released a few days ago 😅

No, to me it's really more about the idiomaticity (sp?), what to expect in a Laravel based library.

Reply:
I took your comment about not booting a little too far. You meant just avoid booting GraphQL. Resolving a class through the container wouldn't trigger booting GraphQL. The app is already booting anyway so using the container there is fine.
Silly me. 😬